src/metabase/cmd/load_from_h2.clj

```diff
@@ -32,4 +35,5 @@
   ([h2-filename]
    (let [h2-filename    (str h2-filename ";IFEXISTS=TRUE")
          h2-data-source (copy.h2/h2-data-source h2-filename)]
-     (copy/copy! :h2 h2-data-source (mdb/db-type) (mdb/data-source)))))
+     (copy/copy! :h2 h2-data-source (mdb/db-type) (mdb/data-source))
+     (search.index/reset-tracking!))))
```

Comment:
this is a bit weird as it's "postgres" and not just generic index

Reply:
Fixed in the refactor